packages/runtime/src/state/actions.ts

```diff
@@ -84,6 +85,8 @@ export const ActionTypes = {
   SET_LOCALE: 'SET_LOCALE',
 
   SET_LOCALIZED_RESOURCE_ID: 'SET_LOCALIZED_RESOURCE_ID',
+
+  UPDATE_API_CLIENT_CACHE: 'UPDATE_API_CLIENT_CACHE',
```

Comment:
New action to update the client cache.

Reply:
It's dispatched here: https://github.com/makeswift/makeswift/pull/836/files#diff-574436562edd329137174ba80e6ecdb186f78f8e2b7e6cbc929afe8ede60e4e9R21